packages/core/src/types.ts

```diff
@@ -19,25 +19,18 @@ import {
   UNSUBSCRIBE_TYPE,
   INVALIDATE_TYPE,
   GC_TYPE,
+  OPTIMISTIC_TYPE,
 } from '@rest-hooks/core/actionTypes';
 import type Controller from '@rest-hooks/core/controller/Controller';
+import type { ErrorTypes } from '@rest-hooks/endpoint';
+import type { EndpointUpdateFunction } from '@rest-hooks/core/controller/types';
 
 export type { AbstractInstanceType, UpdateFunction };
 
 export type ReceiveTypes = typeof RECEIVE_TYPE;
 
 export type PK = string;
 
-export interface NetworkError extends Error {
```

Comment:
These seem like breaking changes as well, since they’re not re-exported

Reply:
These were moved to endpoint. All endpoint exports are reexportted. Since this has a normal dep of endpoint it should include that addition